packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -21,6 +29,94 @@ List<Widget> dialogSubHeader(ThemeData theme, String titleText) {
   ];
 }
 
+/// A standardized dialog with help text and buttons `Reset to default`,
+/// `APPLY` and `CANCEL`.
+class DialogWithDefaults extends StatelessWidget {
+  const DialogWithDefaults({
+    super.key,
+    required this.title,
+    required this.child,
+    this.helpText,
+    this.onResetDefaults,
+    this.onApply,
+    this.onCancel,
+    this.dialogWidth,
+  });
+
+  final String title;
+  final String? helpText;
+  final VoidCallback? onResetDefaults;
+  final VoidCallback? onApply;
+  final VoidCallback? onCancel;
+  final Widget child;
+  final double? dialogWidth;
+
+  @override
+  Widget build(BuildContext context) {
+    return DevToolsDialog(
+      title: _DialogWithDefaultsTitle(onResetDefaults: onResetDefaults),
+      content: Container(
+        padding: const EdgeInsets.symmetric(
+          horizontal: defaultSpacing,
+        ),
+        width: dialogWidth ?? defaultDialogWidth,
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            child,
+            if (helpText != null) ...[
```

Comment:
the help text should probably be at the top of the dialog, not bottom. swap order with child

Reply:
Oh actually I see why you did this now, since this was the existing behavior for the filter dialogs. Will this order apply to the new filter dialog you are adding? if not, we should consider passing in `children` instead of `child`. That way the layout order is more flexible

Reply:
Yes, i am ok to have help at the bottom. 
From UX perspective it may be even better as the user first want to be puzzled in order to be interested in help.